metadata.yaml

```
@@ -3,12 +3,12 @@
 name: kafka
 display-name: Apache Kafka
 description: |
-  Kafka is an event streaming platform. This charm deploys and operates Kafka on
+  Apache Kafka is an event streaming platform. This charm deploys and operates Apache Kafka on
   a VM machines environment.
 
   Apache Kafka is a free, open source software project by the Apache Software Foundation. 
-  Users can find out more at the [Kafka project page](https://kafka.apache.org/).
-summary: Charmed Kafka Operator
+  Users can find out more at the [Apache Kafka project page](https://kafka.apache.org/).
+summary: Charmed Apache Kafka Operator
```

`Operator` or `operator`?

---

Ideally, it should be a lower-case operator.
```suggestion
summary: Charmed Apache Kafka operator
```